fix(curriculum): clarify RangeError example in debugging lecture (#63964)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/lecture-debugging-techniques/6733aa9b006d29f4d11307a5.md

The last common error we will look at is the `RangeError`.

A `RangeError` happens when your code tries to use a value that’s outside the range of what JavaScript can handle. Here is an example of assigning an invalid value to an array's length:

```js
const arr = [];
arr.length = -1; 
```

> Since `-1` is not a valid value for an array's `length` (the length must be a non-negative integer), assigning it will result in a `RangeError`.

Reviewer: ```suggestion
Since an array's length has to be a non-negative integer, setting it to `-1` triggers a `RangeError`.
```
